functions/Set-DbaAgReplica.ps1

```diff
@@ -1,3 +1,4 @@
+using namespace System.Collections.Generic;
```

Comment:
Thanks for the PR, Ben! Unfortunately, `using` is not valid PSv3 syntax and we cannot drop v3 support. Is there a way you can do this using Add-Type, perhaps?

Reply:
Is it required?

Reply:
Is it required?

> Is it required?

It's not required, but makes this:

```
[List[IList[string]]]::new()
```

become this:

```
[System.Collections.GenericList[System.Collections.GenericIList[string]]]::new()
```

> Is there a way you can do this using Add-Type, perhaps?

Maybe! I'll play around with it. Worst case, I change to what I have above (i.e. full-pathing the type name). Thanks for the suggestion!

Reply:
fixed it, that was fun! it's so rare to have to use such complex types. i'll definitely use this as a template when I have to do something similar in the future, thankks.

Reply:
I also learned a lot in re: using complex types in getting the code to where it was/is.